app/lib/apicast/curl_command_builder.rb

```diff
@@ -17,7 +17,9 @@ def command
         credentials = proxy.authentication_params_for_proxy
         extheaders = ''
 
-        uri = Addressable::URI.parse(base_endpoint)
+        uri = uri_base_endpoint
+        return 'curl invalid-uri' unless uri
```

Comment:
Can we maybe render an empty string instead? It doesn't seem that the invalid URI is something to report in the curl command, especially like this without a distinguishable styling of the text.

```suggestion
        return unless uri
```